src/main/resources/log4j.properties

```diff
@@ -0,0 +1,12 @@
+log4j.rootLogger=DEBUG,FILE
+
+log4j.appender.FILE=org.apache.log4j.DailyRollingFileAppender
+log4j.appender.FILE.Append = true
+log4j.appender.FILE.DatePattern = '.'yyy-MM-dd
```

Comment:
There seems to be a typo here: '.'yyy-MM-dd could be corrected to
 '.dd-MM-yyyy' to match the bulgarian date format.